components/03-modules/newsletter/_newsletter.scss

```diff
@@ -0,0 +1,94 @@
+$newsletter__font-size                : $font-size-base !default;
+$newsletter__font-size--agreements    : $font-size-small !default;
+$newsletter__field-border-radius      : 100px !default;
+$newsletter__checkbox-label-opacity   : 0.7 !default;
+$newsletter__line-height              : 1.2 !default;
+$newsletter__input-background         : #717171 !default;
+$newsletter__input-color              : $white !default;
+$newsletter__input-color--placeholder : rgba(255, 255, 255, 0.7) !default;
+$newsletter__heading-color            : $white !default;
+$newsletter__controls-height          : 32px !default;
+$newsletter__checkbox-bg-color        : $white !default;
+$newsletter__heading-gap              : $spacer--medium !default;
+
+.newsletter {
+    &__heading {
+        line-height: $newsletter__line-height;
+        margin-right: 20px;
+        color: $newsletter__heading-color;
+        font-family: $font-family-base;
+        height: auto;
+        font-size: $newsletter__font-size;
+        font-weight: $font-weight-bold;
```

Comment:
is it ok?

Reply:
yup, the way that different font are loading has changed a little (in font-face) and it's ok on FF too, I checked it. ;)